### ISSUE TITLE
'packages' fails to include complete packages since 0.11.0.0

### ISSUE THREAD

**Issue by lukaszgo1** [py2exe/py2exe#113]
To reproduce freeze 
[this reproducer](https://github.com/py2exe/py2exe/files/7544517/py2exe_issue_2.zip) with py2exe version 0.10.4.1 and then with version 0.11.0.0, start the created binaries and compare the output.
The build frozen with 0.11.0.0 shows the following error:
```
Traceback (most recent call last):
  File "main.py", line 2, in <module>
    pkg.import_by_name("named_module").disp()
  File "D:\py2exe_issue_2\pkg\pkg_content\__init__.py", line 5, in import_by_name
    mod = importlib.import_module("pkg.pkg_content.%s"%name, package="pkg.pkg_content")
  File "C:\Python37\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'pkg.pkg_content.named_module'

```
Inspecting library.zip shows that indeed pkg.pkg_content.named_module is missing even though entire `pkg` package is forcefully included in the setup file.

**Comment by albertosottile** [comment]
**TL;DR**: replace `"packages": ["pkg"]` with `"packages": ["pkg.pkg_content"]`. In general, use the options in `setup.py` to specify only imports not handled correctly by the module finder.

Rather long explanation:

This is the "expected behavior" with the new module finder, to some extent. The `packages` keyword in the `setup.py` script now refers just to the packages themselves, and not to "a package, with all its modules, and all its subpackages, with all their modules (iterated recursively)". 

This is what CPython ModuleFinder does (you can verify this by running it it on your snippet) and I think it's the sanest approach. For example, this allows to freeze applications with only a subset of large GUI libraries (e.g. `PySide2.QtCore, PySide2.QtWidgets`) without having to bundle all the Qt libraries (hundreds of MB). Before the new module finder, one had to include the full `PySide2`, then manually delete the unneeded libraries from the frozen bundle.

That being said, let's go back to your bug. You import `pkg.pkg_content.named_module` in a way that the module finder does not recognize. For this reason, you had always needed to instruct `py2exe` to pick up your package. Before, you did this on the parent package, but now module finder is capable of doing a large part of the job for you on its own. You have to instruct it just for the packages/modules not imported in a standard way, in this case `pkg.pkg_content.named_module`.

Hence, you can easily solve this by adjusting your `setup.py` script, either by embedding the full package ( `"packages": ["pkg.pkg_content"]`) or by adding the missing module (`"includes": ["pkg.pkg_content.named_module"]`), whatever it feels more convenient to you.

**Comment by lukaszgo1** [comment]
Thanks - what you've proposed worked both for my MWE and for the issue which has been introduced in NVDA after we updated to py2exe 0.11.0.0. I'm closing this issue.
